linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -1905,10 +1909,10 @@ where
         let ownership = &recipient_chain.manager.get().ownership;
         assert!(
             ownership
-                .super_owners
+                .owners
                 .contains_key(&recipient_key_pair.public().into())
-                && ownership.super_owners.len() == 1
-                && ownership.owners.is_empty()
+                && ownership.super_owners.is_empty()
+                && ownership.owners.len() == 1
```

Comment:
Is this the test that's supposed to exercise the changed functionality? It's called `test_handle_certificate_to_active_recipient`

Reply:
I guess you could say that, since it is sensitive to whether the change works as intended - although, of course, it doesn't really exercise it _directly_.